platform/platform-commands/src/main/java/org/codice/ddf/commands/platform/ConfigStatusCommand.java

```diff
@@ -0,0 +1,83 @@
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.commands.platform;
+
+import java.io.IOException;
+import java.io.PrintStream;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collection;
+
+import org.apache.felix.gogo.commands.Command;
+import org.codice.ddf.configuration.store.ConfigurationStatusService;
+import org.fusesource.jansi.Ansi;
+import org.fusesource.jansi.Ansi.Attribute;
+
+@Command(scope = PlatformCommands.NAMESPACE, name = "configstatus", description = "Lists import status of configuration files.")
+public class ConfigStatusCommand extends PlatformCommands {
+
+    private String configDirectory;
+
+    static final String SUCCESSFUL_IMPORT_MESSAGE = "All config files in [%s] imported successfully.";
+
+    static final String FAILED_IMPORT_MESSAGE = "Failed to import file [%s]. Moved to [%s].";
+
+    private ConfigurationStatusService configStatusService;
+
+    public ConfigStatusCommand(ConfigurationStatusService connfigStatusService,
+            String configDirectory) {
+        this.configStatusService = connfigStatusService;
+        this.configDirectory = configDirectory;
+    }
+
+    @Override
+    protected Object doExecute() {
+        PrintStream console = getConsole();
+        try {
+            Collection<Path> failedImports = getFailedImports();
+
+            if (failedImports.isEmpty()) {
+                outputMessage(console, Ansi.Color.GREEN,
+                        String.format(SUCCESSFUL_IMPORT_MESSAGE, configDirectory));
+            } else {
+                for (Path failedImport : failedImports) {
+                    Path configFile = Paths.get(configDirectory).resolve(failedImport.getFileName());
+                    outputMessage(console, Ansi.Color.RED,
+                            String.format(FAILED_IMPORT_MESSAGE, configFile.toString(), failedImport.toString()));
+                }
+            }
+        } catch (Exception e) {
+            String message = "An error was encountered while executing this command. "
+                    + e.getMessage();
+            outputMessage(console, Ansi.Color.RED, message);
+        }
+
+        return null;
+    }
+
+    private Collection<Path> getFailedImports() throws IOException {
+        return configStatusService.getImportStatus();
+    }
+
+    private void outputMessage(PrintStream console, Ansi.Color color, String message) {
```

Comment:
Like this method. Maybe we should move it up to the base class and have two different ones, `outputMessage` and `outputErrorMessage`? That way the code can be re-used by the other platform commands we'll have to write and we won't need to hard code the green and red colors everywhere. We could also move `getConsole()` to the base class and have the output methods call it when needed.